### ISSUE TITLE
MPV stopped working because of -4 argument

### ISSUE THREAD

**Issue by hjri** [hydrusnetwork/hydrus#1544]
### Hydrus version

527

### Qt major version

Qt 6

### Operating system

Linux (specify distro and version in comments)

### Install method

Third party (AUR, Docker, Chocolatey, etc. Specify in comments)

### Install and OS comments

Arch Linux, AUR, KDE6/Wayland

### Bug description and reproduction

v572, linux, source

opening files with mpv viewer renders nothing and produces this:
```
ValueError
('Invalid value for mpv parameter', -4, (<MpvHandle object at 0x756c5f85a180>, <mpv.LP_MpvNode object at 0x756c5f854830>, <mpv.LP_MpvNode object at 0x756c5f854710>))
Traceback (most recent call last):
  File "/opt/hydrus/hydrus/client/gui/QtPorting.py", line 1320, in eventFilter
    event.Execute()
  File "/opt/hydrus/hydrus/client/gui/QtPorting.py", line 1293, in Execute
    self._fn( *self._args, **self._kwargs )
  File "/opt/hydrus/hydrus/client/gui/canvas/ClientGUICanvas.py", line 1231, in SetMedia
    self._media_container.SetMedia( self._current_media, maintain_zoom, maintain_pan )
  File "/opt/hydrus/hydrus/client/gui/canvas/ClientGUICanvasMedia.py", line 2151, in SetMedia
    self._MakeMediaWindow()
  File "/opt/hydrus/hydrus/client/gui/canvas/ClientGUICanvasMedia.py", line 1524, in _MakeMediaWindow
    self._media_window = CG.client_controller.gui.GetMPVWidget( self )
                         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/hydrus/hydrus/client/gui/ClientGUI.py", line 7366, in GetMPVWidget
    mpv_widget = ClientGUIMPV.MPVWidget( parent )
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/hydrus/hydrus/client/gui/canvas/ClientGUIMPV.py", line 293, in __init__
    self._player.loadfile( self._black_png_path )
  File "/usr/lib/python3.11/site-packages/mpv.py", line 1329, in loadfile
    self.command('loadfile', filename.encode(fs_enc), mode, MPV._encode_options(options))
  File "/usr/lib/python3.11/site-packages/mpv.py", line 1227, in command
    _mpv_command_node(self.handle, ppointer, out)
  File "/usr/lib/python3.11/site-packages/mpv.py", line 142, in raise_for_ec
    raise ex
ValueError: ('Invalid value for mpv parameter', -4, (<MpvHandle object at 0x756c5f85a180>, <mpv.LP_MpvNode object at 0x756c5f854830>, <mpv.LP_MpvNode object at 0x756c5f854710>))
```

**Comment by Segafox** [comment]
Can confirm, same Linux distro, Hydrus v571
Seems like latest Arch mpv release (v0.38.0-2) broke something (again)

As a temporary solution, downgrading to v0.37.0-3 seems to fix it.

**Comment by trashcand69** [comment]
https://github.com/mpv-player/mpv/blob/master/DOCS/interface-changes.rst

>--- mpv 0.38.0 ---
>...
>   - add `index` argument to `loadfile` and `loadlist` commands
>   - move the `options` argument of the `loadfile` command from the third
>     parameter to the fourth (after `index`)

**Comment by internationalcrisis** [comment]
I am using Arch as well and had this issue in April after updating my system. I believe `python-mpv` was out of date at the time, modifying the PKGBUILD to install v1.0.6 fixed it.

This issue may be fixed as the package was updated to v1.0.6, but I could be wrong as it's been some time.

**Comment by hydrusnetwork** [comment]
Hey, thank you for the reports. As @internationalcrisis says, the new 1.0.6 'mpv' python package fixes this. Seems like mpv just changed up their libmpv API for whatever reason. This should be fixed on the release builds, and anyone who runs from source should rebuild their venv with the setup_venv script!

I do not know what flathub, AUR, _et al_ need to do to fix this, but a rebuild of the venv with the new requirements.txt should sort them out.
